src/liblibc/lib.rs

```diff
@@ -227,7 +227,7 @@ pub mod types {
                 pub type rlim_t = u64;
 
                 #[repr(C)]
-                #[derive(Copy)] pub struct glob_t {
+                #[derive(Copy, Clone)] pub struct glob_t {
```

Comment:
I see. Seems like a minor thing to me though. You can still avoid trait impls beyond Copy/Clone if you choose.